pkg/cloudprovider/aws/apis/v1alpha1/provider.go

```diff
@@ -78,6 +79,9 @@ type AWS struct {
 	// required.
 	// +optional
 	MetadataOptions *MetadataOptions `json:"metadataOptions,omitempty"`
+	// BlockDeviceMappings to be applied to provisioned nodes.
+	// +optionals
+	BlockDeviceMappings []*BlockDeviceMapping `json:"blockDeviceMappings,omitempty"`
```

Comment:
Thoughts on making a new type `LaunchTemplate` and inlining?
```
type AWS struct {
  LaunchTemplateName *string `json:"launchTemplate,omitempty"` // Need to backwards incompat later
  LaunchTemplate `json:"inline"`
}
type LaunchTemplate struct {
  	MetadataOptions *MetadataOptions `json:"metadataOptions,omitempty"`
	BlockDeviceMappings []*BlockDeviceMapping `json:"blockDeviceMappings,omitempty"`
}
```

Eventually we should clean up to 
```
type AWS struct {
  LaunchTemplate LaunchTemplate `json:"launchTemplate,omitempty"`
}
```